pl.yaml

```diff
@@ -37,95 +37,95 @@ COUPON_ADDED: Kupon {coupon} został dodany do twojego ekwipunku kuponów.
 COUPON_CLAIM: Odbierz kupon.
 COUPON_CLAIM_NOTHING: Nie masz dostępnych kuponów
 COUPON_INFORMATION:
-COUPON_NOT_VALID: Kupon {coupon} jest nie ważny lub zostal już użyty
-COUPON_NWC_ADDED: The amount of {coins} NWC has been added to your account.
-COUPON_INVENTORY: Coupon Inventory
-COUPON_PLACEHOLDER: Enter coupon's code
-COUPON_REDEEM: Redeem a coupon
-CUSTOMIZE: Customize
-DISABLED: Disabled
-DISCONNECT: Disconnect
-DOWNLOAD_LAUNCHER: Download Launcher
-DOWNLOAD_LAUNCHER_VERSION: Download NosWings Launcher (v.{version})
-DOWNLOAD_LAUNCHER_x64_VERSION: Download x64 version
-DOWNLOAD_LAUNCHER_x86_VERSION: Download x86 version
-DOWNLOAD_LAUNCHER_INFO_MESSAGE: If you have no idea about architecture, please download the x86 version
+COUPON_NOT_VALID: Kupon {coupon} jest już nieważny lub został już użyty
+COUPON_NWC_ADDED: Ilość {coins} NWC została dodana do twojego konta.
+COUPON_INVENTORY: Ekwipunek Kuponów
+COUPON_PLACEHOLDER: Wpisz kod kuponu
+COUPON_REDEEM: Zrealizuj kupon
+CUSTOMIZE: Spersonalizuj
+DISABLED: Wyłączone
+DISCONNECT: Odłącz
+DOWNLOAD_LAUNCHER: Pobierz Launcher
+DOWNLOAD_LAUNCHER_VERSION: Pobierz NosWings Launcher (wersja {version})
+DOWNLOAD_LAUNCHER_x64_VERSION: Pobierz wersję x64
+DOWNLOAD_LAUNCHER_x86_VERSION: Pobierz wersję x86
+DOWNLOAD_LAUNCHER_INFO_MESSAGE: Jeśli nie masz pojęcia jaką architekturę wybrać, wybierz wersję x64
 EMAIL:
-  CONFIRM_NEW: Confirm new email
-  CURRENT: Current email
-  NEW: New email
-  TITLE: Email
-  UPDATE: Update Email
-ENABLED: Enabled
-GAME_ACCOUNTS: Game Accounts
-GENERAL: General
-HOME: Home
-LANGUAGE_CZ: Czech
-LANGUAGE_DE: German
-LANGUAGE_EN: English
-LANGUAGE_ES: Spanish
-LANGUAGE_FR: French
-LANGUAGE_IT: Italian
-LANGUAGE_PL: Polish
-LANGUAGE_TR: Turkish
-LOGIN: Login
-LOGOUT: Logout
-LOGS: Logs
-LOGS_DONATION: Donation Logs
-LOGS_SHOP: Shop Logs
-NO_RECORDS_FOUND: No records found
-NUMBER_OF_CHARACTERS: Number of characters
-PAGINATION_TOTAL_COUNT: Showing from {min} to {max} of {total} entries
+  CONFIRM_NEW: Potwierdź nowy e-mail
+  CURRENT: Aktualny e-mail
+  NEW: Nowy e-mail
+  TITLE: E-mail
+  UPDATE: Zaktualizuj E-mail
+ENABLED: Włączone
+GAME_ACCOUNTS: Konta Gry
+GENERAL: Ogólne
+HOME: Główna
+LANGUAGE_CZ: Czeski
+LANGUAGE_DE: Niemiecki
+LANGUAGE_EN: Angielski
+LANGUAGE_ES: Hiszpański
+LANGUAGE_FR: Francuski
+LANGUAGE_IT: Włoski
+LANGUAGE_PL: Polski
+LANGUAGE_TR: Turecki
+LOGIN: Zaloguj
+LOGOUT: Wyloguj
+LOGS: Historia
+LOGS_DONATION: Historia Dotacji
```

Comment:
```suggestion
LOGS_DONATION: Historia Darowizn
```
"donation" nie jest tym samym co polska "dotacja". Jak już to "donacja" mogłaby być też.